spec/integrations/compatibility_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require "spec_helper"
```

Comment:
`"spec_helper"` loads `"anycable-rails"` but we want to test that `compatibility` works in isolation from any `anycable` code.

And we shouldn't test it in the context of RPC too.

So, the test setup should be:
- Load rails app with action cable
- Load `anycable/rails/compatibility`
- Call Action Cable code.